spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasLayer.scala

```diff
@@ -51,23 +66,80 @@ object KerasLayerSerializer extends ContainerSerializable with TKerasSerializerH
   }
 }
 
+/**
+ * Wrap a torch style layer to keras style layer.
+ * This layer can be built multiple times.
+ * We are supposing the inputshape and the outputshape keep the same in this layer.
+ * @param layer a torch style layer
+ * @return a keras compatible layer
+ */
+class KerasIdentityWrapper[T: ClassTag]
+(val layer: AbstractModule[Activity, Activity, T])(implicit ev: TensorNumeric[T])
+  extends KerasLayer[Activity, Activity, T](null) {
+  if (layer.isKerasStyle()) {
+    throw new RuntimeException(s"We only accept torch layer here, but got: $layer")
+  }
+  override def computeOutputShape(inputShape: Shape): Shape = {
+    inputShape
+  }
+  override def doBuild(inputShape: Shape): AbstractModule[Activity, Activity, T] = layer
+
+  override def allowRebuilt(): Boolean = false
+
+}
+
+/**
+ * Wrap a torch style layer to keras style layer.
+ * This layer can be built multiple times.
+ * @param torchLayer a torch style layer
+ *   i.e If the input data is (2, 3, 4) and 2 is the batch size, you should input: (3, 4) here.
+ * @return a keras compatible layer
+ */
+class KerasLayerWrapper[T: ClassTag]
+(val torchLayer: AbstractModule[Activity, Activity, T],
+    val inputShape: Shape = null,
+    computeOutputShapeFunc: Option[(Shape => Shape)] = None)(implicit ev: TensorNumeric[T])
```

Comment:
@jason-dai function cannot be serialized to `proto` for now, so we may need to think of another way to wrap the activation.

Reply:
There can be three different wrappers:
1) Identity shape
2) Torch layer w/ no computeOutputshape - just call forward
3) Torch layer w/ computeOutputshape defined